ci/long_running_tests/.rayproject/project.yaml

```diff
@@ -0,0 +1,23 @@
+name: long-running-tests
```

Comment:
Seems weird to put the project in a hidden .rayproj folder, why not remove the dot? Users will be confused by it being hidden.

Reply:
good point, I will change it in a follow up PR